src/opentelemetry/launcher/configuration.py

```diff
@@ -360,14 +418,70 @@ def configure_opentelemetry(
 
     logged_attributes.update(resource_attributes)
 
-    for key, value in logged_attributes.items():
-        _logger.debug("%s: %s", key, value)
-
     if log_level <= DEBUG:
         get_tracer_provider().add_span_processor(
             BatchSpanProcessor(ConsoleSpanExporter())
         )
 
+    if metrics_enabled:
+
+        _logger.debug("configuring metrics")
+
+        logged_attributes[
+            "metrics_exporter_endpoint"
+        ] = metrics_exporter_endpoint
+
+        if metrics_exporter_temporality_preference == "DELTA":
+
+            instrument_class_temporality = {
```

Comment:
The way you are configuring things here leads me to believe that a custom "stateless" config option would be straightforward, as we can simply fine tune the aggregation per instrument. Is that correct? Sounds about right?

Reply:
(that's not for the initial support, of course - more of a potential follow up ;) )

Reply:
Right, should address along with expo histogram